### ISSUE TITLE
How to generate images like Class Specific Image Generation section

### ISSUE THREAD

**Issue by Kelvin945** [utkuozbulak/pytorch-cnn-visualizations#2]
I try to generate worm snake sample using your code. but it didn't look as clear as your sample image. You mentioned that  "generated with regular CNNs with optimizing the input (rather than the model weights)". May I know the input setting?

**Comment by utkuozbulak** [comment]
Hello,

That example uses an image generated from a random uniform distribution between 0 and 255.
This line:
`self.created_image = np.uint8(np.random.uniform(0, 255, (224, 224, 3)))`

Running the code directly produces following images for me (AlexNet - flamingo):
Iter 0:
![iteration_1](https://user-images.githubusercontent.com/19213588/33886372-be448e76-df89-11e7-9ace-03c786718990.jpg)

Iter 100:
![iteration_100](https://user-images.githubusercontent.com/19213588/33886381-c2b26938-df89-11e7-9dfe-72996de1d58a.jpg)

Iter 200:
![iteration_199](https://user-images.githubusercontent.com/19213588/33886301-7787cb1a-df89-11e7-8f70-eba573807aeb.jpg)

I used VGG19 to generate the snake and the spider. I think I also incorporated a Gaussian blur for that (unrelated to the generation technique, just a way to improve the quality of image). 

Perhaps I should include the code for VGG19, incorporating with blurring as well. Though, technically, it is just passing VGG19 as the model instead of AlexNet and blurring the generated image every *n* iteration.

Would it be helpful ?
